cpp/models/abm/world.cpp

```diff
@@ -103,6 +101,19 @@ void World::migration(TimePoint t, TimeSpan dt)
             }
         }
     }
+    // check if a person makes a trip
+    size_t num_trips = m_migration_data.get_trips().size();
+    if (num_trips != 0) {
+        while (m_migration_data.get_next_trip_time() < t + dt && m_migration_data.get_current_index() < num_trips) {
+            auto& trip   = m_migration_data.get_next_trip();
+            auto& person = m_persons[trip.person_id];
+            if (!person->is_in_quarantine() && person->get_location_id() == trip.migration_start) {
+                Location& target = get_individualized_location(trip.migration_target);
```

Comment:
reminder to create a new issue to make the trip destination choice more flexible, e.g. go to a specific location, not to a location type.